src/Tizen.NUI/src/public/Layouting/LayoutGroup.cs

```diff
@@ -26,33 +26,40 @@ namespace Tizen.NUI
     /// <summary>
     /// [Draft] LayoutGroup class providing container functionality.
     /// </summary>
-    internal class LayoutGroup : LayoutItem, ILayoutParent
+    public class LayoutGroup : LayoutItem, ILayoutParent
     {
-        protected List<LayoutItem> _children{ get;} // Children of this LayoutGroup
+        /// <summary>
+        /// [Draft] List of child layouts in this container.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        protected List<LayoutItem> LayoutChildren{ get;} // Children of this LayoutGroup
 
         /// <summary>
         /// [Draft] Constructor
         /// </summary>
+        /// <since_tizen> 6 </since_tizen>
         public LayoutGroup()
         {
-            _children = new List<LayoutItem>();
+            LayoutChildren = new List<LayoutItem>();
         }
 
         /// <summary>
         /// [Draft] Constructor setting the owner of this LayoutGroup.
         /// </summary>
         /// <param name="owner">Owning View of this layout, currently a View but may be extending for Windows/Layers.</param>
+        /// <since_tizen> 6 </since_tizen>
         public LayoutGroup(View owner) : base(owner)
         {
-            _children = new List<LayoutItem>();
+            LayoutChildren = new List<LayoutItem>();
         }
 
-          /// <summary>
+        /// <summary>
         /// From ILayoutParent.<br />
         /// </summary>
+        /// <since_tizen> 6 </since_tizen>
         public virtual void Add(LayoutItem childLayout)
```

Comment:
Parameters of method must have valid documentation. Method has 1 parameter, 0 are documented.